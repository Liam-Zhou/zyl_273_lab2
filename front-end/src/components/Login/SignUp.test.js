
import React from 'react';
import SignUp from './SignUp';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

it('sign up page correctly', () => {
  const component = mount( <SignUp ></SignUp>);
  expect(component).toMatchSnapshot();
});

