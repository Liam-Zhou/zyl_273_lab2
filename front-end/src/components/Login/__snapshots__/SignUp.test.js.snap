// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sign up page correctly 1`] = `ReactWrapper {}`;
